src/components/course/data/courseLoader.js

```diff
@@ -99,7 +103,7 @@ export default function makeCourseLoader(queryClient) {
           subscriptionLicense,
           licenseRequests,
           couponCodeRequests,
-          couponCodesCount: couponCodeAssignments.length,
+          couponCodesCount: couponCodeRedemptionCount,
```

Comment:
Bug fix. With the frontend performance refactor, we no longer considered multi-use codes (i.e., individual codes that have multiple redemptions available).